client/components/popover/index.jsx

```diff
@@ -118,12 +120,17 @@ class Popover extends Component {
 	}
 
 	componentDidUpdate( prevProps, prevState ) {
-		const { isVisible } = this.props;
+		const { isVisible, onShow, focusOnShow } = this.props;
 
 		if ( ! prevState.show && this.state.show ) {
 			this.bindEscKeyListener();
 			this.bindDebouncedReposition();
 			bindWindowListeners();
+			onShow();
```

Comment:
Note: this is unrelated to the current issue, but the `onShow` prop is documented in components readme, but it was not used/called anywhere.

Reply:
I assume this is an optional handler passed by the parent component.

Reply:
@gwwar yes, but passing the handler would do nothing since it was never executed.

Reply:
Ah I see what you're saying now. Let's leave this for a different janitorial. We can do an audit on usages, and if no one actually is using this, we can remove the prop and update the readme.

Reply:
Sure, removed.